socorro/middleware/search_common.py

```diff
@@ -278,7 +278,8 @@ def fix_date_parameter(self, parameters):
                         msg='date must have a prefix operator'
                     )
                 today = datetimeutil.utc_now()
-                if param.value > today:
+                # We add a safety margin to not be too strict.
+                if param.value > today + datetime.timedelta(days=1):
```

Comment:
uh? now I'm royally confused. 
So SuperSearch allows future dates? How can that be? We've had so many errors because it fails hard when the index doesn't exist because the user "accidentally" typed in a future date. So, is it that it allows it differently depending on how far in the future it is?

So how does Wayne get his errors? I've never seen them myself and I don't even know how to fake my OS's clock so I can't pretend "now" is in an hours time. 

I get a feeling that ALL of this can be avoided if avoid relying on the client to set the date. E.g. render the `<input name="start_date" value="{{ server_now_date.isodate() }}">` or something.